styles/styles.css

```
@@ -738,13 +755,224 @@ body {
   font-size: 20px;
   font-weight: 700;
   line-height: 26px;
-  text-align: center;
+  text-align: left;
+}
+
+.infrastructure {
+  position: relative;
+  display: block;
+  padding-left: 39px;
+}
+
+.infrastructure-mark {
+  position: absolute;
+  width: 20px;
+  height: 20px;
+  background-color: #ffffff;
+  border-radius: 4px;
+  top: 0;
+  left: 0;
+}
+
+.infrastructure:focus .infrastructure-mark {
+  outline: 3px solid #83b3d3;
+  background-color: rgba(255, 255, 255, 0.6);
+}
+
+.infrastructure:active .infrastructure-mark {
+  background-color: rgba(255, 255, 255, 0.3);
+  outline: none;
+}
+
+.infrastructure:hover .infrastructure-mark {
+  background-color: rgba(255, 255, 255, 0.6);
+}
+
+.infrastructure-input[type="checkbox"]:checked + .infrastructure-mark {
+  position: absolute;
+  content: "";
+  height: 20px;
+  width: 20px;
+  background-image: url("/images/tick.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.type-home {
+  position: relative;
+  display: block;
+  padding-left: 39px;
+}
+
+.type-home-mark {
+  position: absolute;
+  width: 20px;
+  height: 20px;
+  background-color: #ffffff;
+  border-radius: 50%;
+  top: 0;
+  left: 0;
+}
+
+.type-home:focus .type-home-mark {
+  outline: 3px solid #83b3d3;
+  background-color: rgba(255, 255, 255, 0.6);
+}
+
+.type-home:active .type-home-mark {
+  background-color: rgba(255, 255, 255, 0.3);
+  outline: none;
+}
+
+.type-home:hover .type-home-mark {
+  background-color: rgba(255, 255, 255, 0.6);
+}
+
+.type-home-input[type="radio"]:checked + .type-home-mark::after {
+  position: absolute;
+  content: "";
+  height: 10px;
+  width: 10px;
+  border-radius: 50%;
+  background: #3f5e72;
+  top: 5px;
+  left: 5px;
+}
+
+/*RANGE*/
+
+.catalog-filter-group-price {
+  width: 287px;
+  margin-right: 81px;
+}
+.range-scale {
+  position: relative;
+  top: 32px;
+  height: 4px;
+  background-color: rgba(255, 255, 255, 0.3);
+}
+
+.range-bar {
+  position: absolute;
+  height: 4px;
+  background-color: #ffffff;
+}
+
+.button-toggle {
+  position: absolute;
+  width: 20px;
+  height: 20px;
+  background-color: #ffffff;
+  border-radius: 5px;
+  cursor: pointer;
+  border: none;
+}
+
+.button-toggle:focus {
+  outline: 3px solid #83b3d3;
+}
+
+.button-toggle:active {
+  outline: 2px solid #83b3d3;
+  outline: none;
+}
+
+.button-toggle:hover {
+  background-color: #ffffff;
+}
+
+.toggle-min {
+  top: -8px;
+  left: 0px;
+}
+
+.toggle-max {
+  top: -8px;
+  right: 0;
+}
+
+.range-wrapper-inputs {
+  display: flex;
+}
+
+.range-input {
+  padding-top: 14px;
+  padding-bottom: 11px;
+  padding-left: 19px;
+  padding-right: 19px;
+  background-color: transparent;
+  border: 2px solid #ffffff;
+  margin-top: 34px;
+  color: #ffffff;
+  font-family: inherit;
+  font-size: 18px;
+  font-weight: 400;
+  line-height: 23px;
+}
+
+.range-input{
```

```suggestion
.range-input {
```
пройдись везде. нужно сделать чтобы везде было красиво